wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/vfs/impl/file/event/VfsEventService.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.vfs.impl.file.event;
+
+import com.google.common.annotations.Beta;
+
+import org.slf4j.Logger;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static java.util.concurrent.Executors.newSingleThreadExecutor;
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * @author Dmitry Kuleshov
```

Comment:
Honestly, I don't see the necessity in describing that abstract class containing one trivial operation but if you do really need that description I do not mind if you add it by yourself. I foresee that you will not see the description for a few more classes so consider adding description to `org.eclipse.che.api.vfs.impl.file.event.HiEventDetectorManager`,
`org.eclipse.che.api.vfs.impl.file.event.LoEventListener`,
`org.eclipse.che.api.vfs.impl.file.event.LoEventQueueHolder`, 
and 
`org.eclipse.che.api.vfs.impl.file.event.HiEventBroadcaster` as well.